src/main/java/com/epam/poc/pageObjects/homepage/PromoCarouselObject.java

```diff
@@ -8,6 +8,10 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
+import java.net.MalformedURLException;
+import java.net.URL;
+
+
```

Comment:
You forgot to remove un-use import libraries, you can take note and remove it on the next test cases implementation.
You can merge now.